tests/bin/cli/update-account-template.test.js

```
@@ -0,0 +1,93 @@
+const fs = require("fs");
```

ESLint

[error] 'fs' is assigned a value but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)